mlflow/store/tracking/gateway/abstract_mixin.py

```diff
@@ -22,7 +22,6 @@ def create_secret(
         secret_name: str,
         secret_value: str,
```

Comment:
@BenWilson2 I think we should update the type hint here to str | dict[str, str]?